pymatbridge/matlab_magic.py

```diff
@@ -211,6 +211,8 @@ def matlab(self, line, cell=None, local_ns=None):
             else:
                 e_s = "There was an error running the code:\n %s"%code
                 result_dict = self.eval(code)
+        except MatlabInterperterError:
+            raise
         except:
```

Comment:
Oh, and we should also wrap `OSErrors` from `subprocess.Popen` as well. That's the part that spawns the matlab process.